internal/archtest/ulid_only_test.go

```
@@ -0,0 +1,89 @@
+				t.Errorf("gen_random_uuid() in migration %s:%d — the DB must not mint random identifiers (non-deterministic under replay); mint a ULID in Go", e.Name(), line)
+			}
+			// A uuid-typed column: "<name> uuid" optionally followed by
+			// constraints. Word-boundary match avoids substrings.
+			for _, tok := range strings.Fields(lower) {
+				if tok == "uuid" || tok == "uuid," {
+					t.Errorf("uuid-typed column in migration %s:%d (%q) — identifiers are text ULIDs (F-15)", e.Name(), line, text)
+					break
+				}
+			}
+		}
+		f.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `f.Close` is not checked

(errcheck)